docs/360.html

```diff
@@ -23,17 +21,12 @@
   </script>
 
   <script type="text/javascript">
-    window.addEventListener('load', function(){
-
-      var cld = cloudinary.Cloudinary.new({ cloud_name: 'demo' });
+    window.addEventListener('load', function() {
 
-      player = cld.videoPlayer('player', {
```

Comment:
@e1adn edited the script but not the visible code example, see here below (line ~70) and in all other example pages.

Reply:
done